session/bootstrap.go

```diff
@@ -1193,9 +1204,13 @@ func syncUpgradeState(s Session) {
 }
 
 func syncNormalRunning(s Session) {
-	_, err := ddl.ResumeAllJobsBySystem(s)
+	jobErrs, err := ddl.ResumeAllJobsBySystem(s)
 	if err != nil {
-		logutil.BgLogger().Fatal("upgrade pause all jobs failed", zap.Error(err))
+		logutil.BgLogger().Warn("[Upgrading] unexpected error to resume all paused jobs: ", zap.Error(err))
+	}
+
+	for _, e := range jobErrs {
+		logutil.BgLogger().Warn("[Upgrading] unable to resume the job, error: ", zap.Error(e))
```

Comment:
```suggestion
		logutil.BgLogger().Warn("[upgrading] unexpected error to resume all paused jobs: ", zap.Error(err))
	}

	for _, e := range jobErrs {
		logutil.BgLogger().Warn("[upgrading] unable to resume the job, error: ", zap.Error(e))
```

These should be in lower case.